gax/src/main/java/com/google/api/gax/batching/BatcherImpl.java

```diff
@@ -328,11 +346,12 @@ private Batch(
       this.batcherStats = batcherStats;
     }
 
-    void add(ElementT element, SettableApiFuture<ElementResultT> result) {
+    void add(ElementT element, SettableApiFuture<ElementResultT> result, long throttledTime) {
```

Comment:
This is an InternalApi, but it says that it is for use by google-cloud-java clients, so it is a breaking chagne for them. Can we make it in anon-breaking way (adding a method overload instead of directly changing the method signature).

Reply:
this method is on a private class, so it doesnt leak to clients